app/models/comment.rb

```diff
@@ -332,6 +336,24 @@ def self.gmail_parsed_mail(mail_doc)
     }
   end
 
+  def self.outlook_parsed_mail(mail_doc)
+    separator = mail_doc.inner_html.match(/(.+)(<div id="appendonsend"><\/div>)(.+)/m)
+    if separator.nil?
+      comment_content = mail_doc
+      extra_content = nil
+    else
+      body_message = separator[1].match(/(.+)(<body dir="ltr">)(.+)/m)
+      comment_content = Nokogiri::HTML(body_message[3])
+      trimmed_message = separator[3].match(/(.+)(<\/body>)(.+)/m)
+      extra_content = Nokogiri::HTML(trimmed_message[1])
+    end
+
+    {
+      "comment_content" => comment_content,
```

Comment:
Please write these in this notation: `comment_content: comment_content`.

Reply:
Would you like me to change it for the other parsing methods as well?

Reply:
Yeah please change those also.

Reply:
@Souravirus I have made the changes. Can I also write the integration tests on this PR?

Reply:
Yeah nice work. You can write the integration tests related to this if you have time to do so. Thank you!!

Reply:
I've been trying to write the integration tests but I keep encountering this issue:
![image](https://user-images.githubusercontent.com/44309027/49857604-9e831280-fdb8-11e8-8fc0-91d69fc2c14f.png)

I have added my user to the users fixture and can't seem to figure out why this happens.

Reply:
@Souravirus could you take a look? I think I have finished the tests.